Mill/PaStepVar.pas

```diff
@@ -0,0 +1,1246 @@
+   Ini_Display_Window_Resolution             ,
+   Ini_Display_XY                            ,
+   Ini_Display_Console_Window_Size           ,
+   Ini_Display_Console_XY                    ,
+   Ini_Display_Console_Popup_Window_Size     ,
+   Ini_Display_Console_Popup_XY              ,
+   Ini_Display_Power_On_Time                 ,
+   Ini_Display_Power_Off_Time                ,
+   Ini_Display_Off_Delay                     ,
+   Ini_Display_DayCode                       ,
+   Ini_Light_Delay                           ,
+   Ini_Auto_DXF_Start_XY                     ,
+   Ini_Auto_DXF_Comment_XY                   ,
+   DXF_Comment_XY                            ,
+   Ini_Simulation_Resolution                 ,
+   Ini_Simulation_Draw_Circle_Tolerance      ,
+   Ini_Probe_3D_Type                         ,
+   Ini_Probe_3D_Slow                         ,
+   Ini_Probe_3D_Fast                         ,
+   Ini_Probe_Tool_Set_Type                   ,
+   Ini_Probe_Tool_Set_X_Safe                 ,
+   Ini_Probe_Tool_Set_Y_Safe                 ,
+   Ini_Vector_Color_String                   ,
+   Ini_Vector_Color_First_String             ,
+   Ini_Line_Z0_1_Color_String                ,
+   Ini_Line_Z0_2_Color_String                ,
+   Ini_Line_Z_Positive_1_Color_String        ,
+   Ini_Line_Z_Positive_2_Color_String        ,
+   Ini_Line_Z_Negative_1_Color_String        ,
+   Ini_Line_Z_Negative_2_Color_String        ,
+   Ini_Arc_Z0_1_Color_String                 ,
+   Ini_Arc_Z0_2_Color_String                 ,
+   Ini_Arc_Z_Positive_1_Color_String         ,
+   Ini_Arc_Z_Positive_2_Color_String         ,
+   Ini_Arc_Z_Negative_1_Color_String         ,
+   Ini_Arc_Z_Negative_2_Color_String         ,
+   Ini_Circle_Z0_Color_String                ,
+   Ini_Circle_Z_Positive_Color_String        ,
+   Ini_Circle_Z_Negative_Color_String        ,
+   Ini_Circle_Disabled_Color_String          ,
+   Ini_Force_Z_To_Min_Color_String           ,
+   Ini_Slightly_Above_Zero_Color_String      ,
+   Ini_G0_Color_String                       ,
+   Ini_G0_Error_Color_String                 ,
+   Ini_Start_Color_String                    ,
+   Ini_Stop_Color_String                     ,
+   Ini_Tool_Color_String                     ,
+   Ini_Material_Color_String                 ,
+   Ini_Bounding_Box_Color_String             ,
+   Ini_Table_Color_String                    ,
+   Ini_Crosshair_Color_String                ,
+   Ini_Material_Size                         ,
+   Ini_Material_Offset                       ,
+   Ini_Table_Size                            ,
+   Ini_Table_Offset                          ,
+   Ini_Data                                  ,
+   Ini_String                                ,
+   Ini_MD5                                   ,
+   Ini_BitSave_MD5                           ,
+   Ini_Bittool_MD5                           ,
+   Ini_Location_MD5                          ,
+   Ini_Pendant_Type                          ,
```

Comment:
@coderabbitai Please give me an example of your suggestion

Reply:
@coderabbitai I see the value in your suggestion, I think it would be good to use records if if the group of variables would be used more than 3 times.  please give me a list of all the variables that could be grouped given this condition